cmd/di_desktop.go

```diff
@@ -278,3 +280,19 @@ func (di *Dependencies) bootstrapUIServer(options node.Options) {
 
 	di.UIServer = uinoop.NewServer()
 }
+
+func (di *Dependencies) bootstrapMMN(options node.Options) {
+	err := di.EventBus.SubscribeAsync("identity-unlocked", func(param string) {
+		info, err := mmn.GetNodeInformation()
+		if err != nil {
+			log.Error("Failed to get NodeInformation for MMN", err.Error())
+			return
+		}
+		info.Identity = param
+		client := mmn.NewMMNClient(options.MMN.Address)
+		client.RegisterNode(*info)
+	})
+	if err != nil {
+		log.Error("Failed to get register to mmn event", err.Error())
```

Comment:
```suggestion
		log.Error("Failed to get register to mmn event", err)
```

Reply:
My suggestion is removing explicit converting to string, it enough to just pass `err` here.